rnwinrt/rnwinrt/react/strings/base.cpp

```diff
@@ -205,22 +205,29 @@ jsi::Value static_enum_data::create(jsi::Runtime& runtime) const
     return jsi::Value(runtime, jsi::Object::createFromHostObject(runtime, std::make_shared<projected_enum>(this)));
 }
 
-jsi::Value static_enum_data::get_value(std::string_view valueName) const
+jsi::Value static_enum_data::get_value(jsi::Runtime& runtime, std::string_view valueName) const
 {
     // TODO: It would also be rather simple to ensure that the array is sorted and do a binary search, however the
     // number of enum elements is typically pretty small, so that may actually be harmful
     auto itr = std::find_if(values.begin(), values.end(), [&](auto& mapping) { return mapping.name == valueName; });
-    if (itr == values.end())
+    if (itr != values.end())
     {
-        return jsi::Value::undefined();
+        return jsi::Value(itr->value);
     }
 
-    return jsi::Value(itr->value);
+    // Look for a matching value (reverse mapping)
+    itr = std::find_if(values.begin(), values.end(), [&](auto& mapping) { return std::to_string(static_cast<int32_t>(mapping.value)) == valueName; });
```

Comment:
Note that casting to `int32_t` is incorrect as the underlying type may be unsigned